src/PIL/PdfParser.py

```diff
@@ -98,9 +98,6 @@ def __eq__(self, other):
             and other.generation == self.generation
         )
 
-    def __ne__(self, other):
-        return not (self == other)
```

Comment:
In my local testing, I found this specific `__ne__` necessary. I think because this inherits from tuple/namedtuple which may have a `__ne__` definition.

Reply:
OK, reverted.